installing/install_config/installation-types.adoc

```diff
@@ -48,6 +63,17 @@ Not all installation options are currently available for all platforms, as shown
 |
 |
 
+|Restricted network
+|✓
```

Comment:
Could just switch the check with `ND` or something.

Reply:
If any changes happen to this page, my inclination is to stick to vague tweaks to the text. I'm not sure if committing to anything else is all that maintainable.